### ISSUE TITLE
`examples/natural_questions.py` fails

### ISSUE THREAD

**Issue by ftesser** [deepset-ai/FARM#520]
**Describe the bug**
Running the `examples/natural_questions.py` script I obtain the error below.
The error is thrown from `model.inference_from_dicts` method.

**Error message**
```
"""
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/fabio/src/git_repositories/FARM/farm/infer.py", line 569, in _create_datasets_chunkwise
    dataset, tensor_names, baskets = processor.dataset_from_dicts(dicts, indices, return_baskets=True)
  File "/home/fabio/src/git_repositories/FARM/farm/data_handler/processor.py", line 361, in dataset_from_dicts
    id_external = self._id_from_dict(d)
  File "/home/fabio/src/git_repositories/FARM/farm/data_handler/processor.py", line 403, in _id_from_dict
    ext_id = try_get(ID_NAMES, d["qas"][0])
  File "/home/fabio/src/git_repositories/FARM/farm/utils.py", line 432, in try_get
    ret = dictionary[key]
TypeError: string indices must be integers
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/fabio/src/git_repositories/FARM/examples/natural_questions.py", line 142, in <module>
    question_answering()
  File "/home/fabio/src/git_repositories/FARM/examples/natural_questions.py", line 135, in question_answering
    result = model.inference_from_dicts(dicts=QA_input, return_json=False) # result is a list of QAPred objects
  File "/home/fabio/src/git_repositories/FARM/farm/infer.py", line 696, in inference_from_dicts
    return Inferencer.inference_from_dicts(self, dicts, return_json=return_json,
  File "/home/fabio/src/git_repositories/FARM/farm/infer.py", line 474, in inference_from_dicts
    return list(predictions)
  File "/home/fabio/src/git_repositories/FARM/farm/infer.py", line 545, in _inference_with_multiprocessing
    for dataset, tensor_names, baskets in results:
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 865, in next
    raise value
TypeError: string indices must be integers

```
**Expected behavior**
NO errors

**Additional context**
It seems that QA_input in the format  "qas" + "context" is not managed here. 
In fact if I try to use the "questions" + "text" keys in the QA_input dictionary, no errors are thrown.
 

**To Reproduce**
run `examples/natural_questions.py` (to save time: the same error is obtained  running the script just from point 9 (https://github.com/deepset-ai/FARM/blob/c6871b67401eef36f9d4310c966063eda10b0dfe/examples/natural_questions.py#L125)
 
**System:**
 - OS: Linux Ubuntu 20.04.1 LTS
 - GPU/CPU: both
 - FARM version: 0.4.7

**Comment by Timoeller** [comment]
Thanks for reporting. I can reproduce the error and created a quick fix in #521 

You can base your experiments off of this branch @ftesser until we find a better solution and merge it into master.

**Comment by ftesser** [comment]
thanks @Timoeller I see your fix also in the master now

**Comment by Timoeller** [comment]
Exactly, we quick-fixed it and will work on a more general solution, as Branden mentioned, later on.

Is the issue fixed for you?

**Comment by ftesser** [comment]
> Is the issue fixed for you?

yes, thanks!

**Comment by tholor** [comment]
Fixed in #521
